nixos/modules/security/pam.nix

```
@@ -1184,6 +1184,19 @@ in
       };
     };
 
+    security.pam.unixAuthArgs = mkOption {
+      default = "sha512";
```

```suggestion
      default = "yescrypt";
```

The actual default on unstable right now.